crashdump/crashdump_backtrace.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash -e
+# Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+LIB_DIR="$(dirname "$(readlink -f "$0")")"
+source "$LIB_DIR/traplib.sh"
+
+# Print long help
+help() 
+{
+cat<<EOF
+Usage:
+Analyse crashdump from the regular release and print on the stdout
+    $(basename $0) --dump <crashdump-file> --tag <git-tag>
+Analyse crashdump from the developer OS build and print on the stdout
+    $(basename $0) --dump <crashdump-file> --dir <build-dir>
+Display help:
+    $(basename $0) --help
+
+Mandatory arguments for regular release:
+    crashdump-file:     The crash dump file grabbed from the phone
+    git-tag:            Git tag of the OS version for mudita os release
+Mandatory arguments for the developer build:
+    crashdump-file:     The crash dump file grabbed from the phone
+    build-dir:          Build directory with the MuditaOS source
+EOF
+    exit 0
+}
+
+#Exit abnormal check
+exit_abnormal() {
+cat<<EOF >&2
+$(basename $0): $1
+Try '$(basename $0) --help' for more information
+EOF
+    exit 1
+}
+
+
+# Check if all programs needed by scripts are installed
+# arg1 .. argn: programs for validate
+# On failure terminate the script and set errno
+validate_req_cmds() {
+    for cmd in $@; do
+        if [ ! $(command -v $cmd) ]; then
+            echo "Error! $cmd is not installed, please use 'sudo apt install $cmd' for install required tool"
```

Comment:
it's worth noticing that sometimes tools have different name as a package than the executable, so package name sometimes != command name.
I'd add also more generic info like:
```
XYZ not installed, please use your package manager to install required tool. E.g. sudo apt install XYZ
```

Reply:
fixed